providers.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>3.113.0"
+    }
+  }
+  required_version = ">= 1.1.0"
+}
```

Comment:
Cred ca acest continut o sa-l punem in main.tf din root folder, iar acolo o sa trebuiasca sa configuram si backend, ca sa se salveze tfstate-ul in azure.

Reply:
On second thought, putem sa facem asta cand ajungem la pipeline-ul de apply, realizez ca nu am scris in ticket despre asta